socs/agents/generator/agent.py

```diff
@@ -306,13 +406,13 @@ def acq(self, session, params=None):
                     session.data['address'] = self.host
 
                     regdata = self.generator.read_cycle()
-
                     if regdata:
                         for reg in regdata:
                             data['data'][reg] = regdata[reg]["value"]
-                            field_dict = {reg: regdata[reg]}
+                            field_dict = {reg: regdata[reg]['value']}
                             session.data['fields'].update(field_dict)
                         session.data.update({'timestamp': current_time})
+                        print(data)
```

Comment:
I assume this was for debugging, so we should remove

Reply:
Done.